+sig/Signal.m

```diff
@@ -106,19 +105,12 @@
     
     p = to(a, b)
     
-    % 'ds = s1.setTrigger(s2)' returns a dependent signal 'ds' which can
-    % only ever take a value of 1. 'ds' initially updates to 1 when 's2' is
-    % set to a truthy value, given that 's1' has a truthy value.
-    % Additional updates of 'ds' take place whenever 's2' is set to a
-    % truthy value, given that 's1' has been "reset" to a truthy value.
+    % 'tr = arm.setTrigger(release)' returns a dependent signal that is true
+    % only when `release` evaluates true after `arm`.
     %
     % Example:
```

Comment:
I know we talked about not doing lengthy examples in code files themselves. I think examples should be code that readers can plug and play. Maybe we could state in the docstring that examples can be found in the `docs` files?